diff --git a/flake.nix b/flake.nix
index b13782ffe4..c85cc7981e 100644
--- a/flake.nix
+++ b/flake.nix
@@ -108,8 +108,8 @@
     // {
       overlay = final: prev:
         let
-          #emacsNgSource = emacsNg-src;
-          emacsNgSource = ./.;
+          emacsNgSource = emacsNg-src;
+          #emacsNgSource = ./.;
           #rust nightly date
           locked-date = prev.lib.removePrefix "nightly-" (prev.lib.removeSuffix "\n" (builtins.readFile ./rust-toolchain));
         in
@@ -224,8 +224,8 @@
               rec {
                 name = "emacsNg-" + version;
                 src = emacsNgSource;
-                #version = builtins.substring 0 7 emacsNgSource.rev;
-                version = "develop";
+                version = builtins.substring 0 7 emacsNgSource.rev;
+                #version = "develop";
 
                 preConfigure = (old.preConfigure or "") + ''
             '';
